Sources/MapboxMaps/Style/Style.swift

```diff
@@ -9,6 +9,7 @@ internal protocol StyleProtocol: AnyObject {
     func removeLayer(withId id: String) throws
     func layerExists(withId id: String) -> Bool
     func setLayerProperties(for layerId: String, properties: [String: Any]) throws
+    func setLayerProperty(for layerId: String, property: String, value: Any) throws
```

Comment:
so you didn't expose this method previously for iOS? This would be a great way to update single style layer property. Would be good to mention in the change log.

BTW, how did you track the public API changes on iOS SDK?

Reply:
emm.. ok, so this API is available in the public-facing Style class previously?

Reply:
ok, thanks for the clarification.